pkg/remotewrite/remotewrite_test.go

```diff
@@ -341,3 +345,84 @@ func TestOutputStaleMarkers(t *testing.T) {
 		assert.True(t, math.IsNaN(markers[i].Samples[0].Value), "it isn't a StaleNaN value")
 	}
 }
+
+func TestOutputStopWithStaleMarkesDisabled(t *testing.T) {
```

Comment:
```suggestion
func TestOutputStopWithStaleMarkersDisabled(t *testing.T) {
```

Though:
- The test is named `WithStaleMarkersDisabled`, yet the config sets `StaleMarkers: true`. :confused:
- And isn't this test a duplicate, considering you test both the `true` and `false` case in `TestOutputStopWithStaleMarkes` below? So I would just remove it.